README.md

```diff
@@ -1,2 +1,243 @@
+| 14:00 - 16:00 | Afternoon Focus Time (_please whisper or use "instant messaging" to communicate_) |
+| 16:00 - 16:30 | Afternoon Break (_moderate chat/noise acceptable_) |
+| 16:30 - 18:30 | "Final Push" Focus Time (_please whisper or use "instant messaging" to communicate_) |
+| 18:30 - 19:00 | "Show & Tell", "Get Help", "Q&A", "Decompress Time" |
+| 19:00 - 20:30 | Meetup, Event or Evening Training (_training space bookable by members_) |
+| 20:30 - 21:00 | Tidy up the space ready for next session. |
+| 21:00 - 23:00 | "Late Shift" Focus Time (_Upstairs only. please whisper or use "instant messaging" to communicate_) |
+| 23:00 - 04:00 | "Night Owls Club" (_for people who work best at night. Upstairs only.  Please apply for a door key. Door must be kept locked except for entry/exit. No Slumber Parties! ;-) Last person out switch off all lights/heaters and set the alarm!_) |
+
+
+## How? (_Proposal_)
+
+Our "***metric***" for this should be a "hybrid" NPS/Productivity score.
+
+> Mark will formulate exactly how to calculate this and capture the data
+  from all members on an on-going basis. This is one of our USPs so getting regular data/feedback is essential.
+
+### TODO
+
++ [ ] **Market _Validation_**
+  + [ ] Create a basic survey (_using your choice of survey tool_)
+  with ***questions*** (_proposed_):
+    + [ ] Where do you get your most valuable work done?
+    + [ ] What time of day are you most productive?
+    + [ ] On a scale of 1 - 5
+    (_where 1 is **disagree strongly** and 5 is **agree strongly**_)
+    + I work best when in a quiet environment.
+    + I regularly wear earphones to block out noise
+    when I'm trying to concentrate on my work.
+    + I find it difficult to get back into my work if I am interrupted.
+    + I like to openly discuss my work with my co-workers and pick their brains without calling a "formal" meeting.
+  + [ ] Collect responses to the survey by using Google Adwords
+  to target people who are searching for "London Co-Working" direct them to the survey.
+  + [ ] Use Google Analytics to track response/dropout rate.
++ [ ] ***Membership Registration/Management*** System (_can be a [**spreadsheet**](https://github.com/dwyl/html-form-send-email-via-google-script-without-server) for MVP with a view to using our [CRM](https://github.com/dwyl/hq/issues/161) after MVP_)
+but ideally would link to the ***payment processing***/verification so it's clear who has paid.
+
++ [ ] Monthly/Pay-as-you-Go **Payment Collection** System (_e.g: GoCardless_)
+
++ [ ] **Desk allocation** system (_so people can **reserve** a **specific desk**
+  if they want to and we **know** when we're "**full**"_)
+
++ [ ] **Check-in** system when people arrive (_also useful for fire safety_)
+
++ [ ] Anonymous ***feedback*** system so people can complain if they feel the need to?
+see: https://github.com/dwyl/hq/issues/183
+
```

Comment:
Please add it in. I'm not _relying_ on it being created `before` we can launch this.